File: Resources/ResponseResult.cs


namespace Resources
{
    public class ResponseResults<T>
    {
        public string Status { get; set; }
        public string Msg { get; set; }
        public T Result { get; set; }
        public string AdditionalInformation { get; set; }

        public ResponseResults()
        {

        }
        public ResponseResults(string Status, string Msg, T Result,string AdditionalInformation="")
        {
            this.Status = Status;
            this.Msg = Msg;
            this.Result = Result;
            this.AdditionalInformation = AdditionalInformation;

        }
      
    }
}

File: MyTimeAtt/SSCode/src/app/shared/responseresult.ts
export class ResponseResult {
    Status: boolean;
    Msg: string;
    Result: any;
    AdditionalInformation:string;
    constructor() {
    }
    public mapResult(res: any)
    {
        this.Status = (res.Status === '1');
        this.Msg = res.Msg;
        this.Result = res.Result;
        this.AdditionalInformation=res.AdditionalInformation;

    }
}
